docs/cli/config-create.md

```diff
@@ -0,0 +1,16 @@
+# 🛠️ Command `config create`
+
+The `config create` command allows you to create a new configuration file for NodeSecure. This command is useful for initializing a custom configuration tailored to your project's specific needs.
+
+## 📜 Syntax
+
+```bash
+nsecure config create [configuration] [options]
+```
+
+## ⚙️ Available Options
+
+| **Name** | **Shortcut** | **Default Value** | **Description**                                                               |
+|----------|--------------|-------------------|-------------------------------------------------------------------------------|
+| `--cwd`  | `-c`         | `false`           | Create the configuration file in the current working directory instead of the default location. |
+
```

Comment:
We can delete this one to only keep config.md I think (and in the README keep the command but redirect to config.md)